Gemfile.lock

```diff
@@ -54,9 +63,15 @@ GEM
       execjs
     coffee-script-source (1.10.0)
     concurrent-ruby (1.0.0)
+    crack (0.4.3)
```

Comment:
I want to share this unnecessary supplemental reading I did after seeing this gem name: [What is crack?](http://www.railstips.org/blog/archives/2009/04/01/crack-the-easiest-way-to-parse-xml-and-json/)